deploy/manifests/dev/us-east-2/tenant/storetheindex/kustomization.yaml

```diff
@@ -8,4 +8,4 @@ resources:
 images:
 - name: storetheindex
   newName: 407967248065.dkr.ecr.us-east-2.amazonaws.com/storetheindex/storetheindex # {"$imagepolicy": "storetheindex:storetheindex:name"}
-  newTag: 20221202100425-3d964311737ecfd308495164c297e5d9a4d348c9 # {"$imagepolicy": "storetheindex:storetheindex:tag"}
```

Comment:
I am curious why are we not using the autogenerated PRs to deploy to dev?

Reply:
I also wonder why. I thought one would appear after a while. When I did not see one after 30+ minutes, I created this.

Reply:
Here is the issue: https://github.com/ipni/storetheindex/actions/runs/3608106922

Reply:
OK, so repo owner just needs to give it permissions.